backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/domain/mapper/ConfirmationCodeMapper.java

```diff
@@ -0,0 +1,32 @@
+package ca.gov.dtsstn.cdcp.api.service.domain.mapper;
+import java.util.List;
+import java.util.stream.StreamSupport;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.springframework.lang.Nullable;
```

Comment:
Use jakarta nullable, not the spring one

Reply:
replaced springframework with jakarta